apps/src/templates/teacherHomepage/StudentHomepage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import RecentCoursesCollapsible from './RecentCoursesCollapsible';
+import CollapsibleSection from './CollapsibleSection';
+import GradientNavCard from './GradientNavCard';
+import shapes from './shapes';
+import i18n from "@cdo/locale";
+
+const StudentHomepage = React.createClass({
+  propTypes: {
+    courses: shapes.courses
+  },
+
+  componentDidMount() {
+    $('#user_hero').appendTo(this.refs.userHero).show();
```

Comment:
I had it this way before, but in the interests of moving towards a single React dom I explicitly moved it here.  It's one of the earliest components I think we'll want to rewrite in React, so leaving it here would at least be a good reminder to do that.  That said, no strong opinion.